app-office/planner/planner-9999.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Project manager for Gnome"
+HOMEPAGE="https://wiki.gnome.org/Apps/Planner"
+if [[ "${PV}" == "9999" ]] ; then
+	EGIT_REPO_URI="https://gitlab.gnome.org/World/planner.git"
+	inherit git-r3
+	SRC_URI=""
+else
+	KEYWORDS="~amd64"
```

Comment:
In theory there should be no reason to lose keywords compared to the release version, but I definitely don't mind to lower the surface a bit and have anyone wanting to use it elsewhere request a keywords later again (once this promotes to a released version). Would tell if there's any such users perhaps :)

Reply:
I know, but honestly the last ebuild is ~9 years old.  I felt like this is like a new package now.